plugins/content_distribution/providers/tvinci/lib/admin/TvinciProfileConfiguration.php

```diff
@@ -110,13 +118,11 @@ protected function tvinciElements()
 		);
 
 		// tag specific configuration
-		$this->addTagItems("ism");
-		$this->addTagItems("ipadnew");
-		$this->addTagItems("iphonenew");
-		$this->addTagItems("mbr");
-		$this->addTagItems("dash");
-		$this->addTagItems("widevine");
-		$this->addTagItems("widevine_mbr");
+		$tvinciDistributionTagsSubForm = new Zend_Form_SubForm(array('DisableLoadDefaultDecorators' => true));
```

Comment:
The one in the populate is in order to actually populate this, the one in tvinciElements is in order to place it correctly in the form (display wise).